src/create_config/process.rs

```diff
@@ -656,6 +658,40 @@ pub fn process_create_config(args: CreateConfigArgs) -> Result<()> {
         );
     }
 
+    if config_data.upload_method == UploadMethod::Pinata {
+        let jwt: String = Input::with_theme(&theme)
+            .with_prompt("What is your Pinata JWT authentication?")
+            .interact()
+            .unwrap();
+
+        let api_gateway = Input::with_theme(&theme)
+            .with_prompt("What is the Pinata gateway for upload?")
+            .default(String::from("default"))
```

Comment:
Looks like this is missing an actual default value.

Reply:
Fixed both default values - thanks!